frontend/package-lock.json

```diff
@@ -27549,7 +27549,7 @@
         "fam-admin-mgmt-api": {
             "version": "file:../client-code-gen/gen/admin-management-api",
             "requires": {
-                "axios": "^0.26.1",
+                "axios": "0.26.1",
```

Comment:
I'm not sure why it generates other changes, but let's make sure this is not changed by accident. thanks!!

Reply:
already looked into it, thanks!